fail2ban/client/fail2banregex.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
@sebres this one is not intended to be executed directly right? Then we don't need shebang here and should remove executable but if set